packages/material-ui/src/InputBase/InputBase.js

```diff
@@ -370,16 +369,13 @@ class InputBase extends React.Component {
         ref: null,
       };
     } else if (multiline) {
-      if (rows && !rowsMax && !rowsMin) {
```

Comment:
We still need this logic:
```jsx
if (rows && !rowsMax) {
```

Reply:
Writing code first thing in the morning is hard =).

Reply:
Maybe we should add a test to catch this?

Reply:
The logic is not obvious. Why is the Textarea not used when `rowsMax` isn't provided? At least, it prevents the Textarea height to go out of control. A test could help.